src/GNDStk/Node/src/write.hpp

```diff
@@ -12,17 +12,20 @@
 //
 // General cases:
 // 1. write(ostream,   FileType)
-// 2. write(file name, FileType) calls 1 after making ostream from file name
+// 2. write(file name, FileType) calls 1 after making ostream from file
 // 3. write(ostream,   string  ) calls 1 after making FileType from string
 // 4. write(file name, string  ) calls 2 after making FileType from string
 
 
 
 // -----------------------------------------------------------------------------
+// Helper
 // write(ostream, int level)
 // For FileType::text
 // -----------------------------------------------------------------------------
 
+private:
```

Comment:
Because I wanted some to be private, and others public?! :-D

Seriously.... The ones that I declared public are the ones I want users to see, whereas the private ones are really just intended to be helpers for the public ones.

I suppose I could put the helpers into `detail::` functions instead, if anyone feels really strongly about this. Here, it was convenient to put them into actual member functions, so that I get the implicit `*this`, and also they're close to the `write()` functions that use them.